### ISSUE TITLE
LoadCoverage as gulp plugin

### ISSUE THREAD

**Issue by hrigner** [SitePen/remap-istanbul#134]
I need to combine several reports into one, and I would love to use loadcoverage for that, with Gulp. Currently it seems that it is "only" gulpRemapIstanbul available for Gulp, right?

**Comment by dylans** [comment]
@hrigner yes, that's the case, though we'd certainly welcome any changes or additions if you or any other Gulp users would like to contribute.
